src/Tizen.NUI/src/public/BaseComponents/TextEditor.cs

```diff
@@ -883,6 +883,84 @@ public string InputUnderline
             }
         }
 
+        /// <summary>
+        /// The Strikethrough property.
+        /// The Strikethrough map contains the following keys :<br />
+        /// <list type="table">
+        /// <item><term>enable (bool)</term><description>Whether the strikethrough is enabled (the default value is false)</description></item>
+        /// <item><term>color (Color)</term><description>The color of the strikethrough (If not provided then the color of the text is used)</description></item>
+        /// <item><term>height (float)</term><description>The height in pixels of the strikethrough (the default value is 1.f)</description></item>
+        /// </list>
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public PropertyMap Strikethrough
```

Comment:
Could change structure property to use 'Strikethrough'?
```csharp
public Strikethrough Strikethrough
```

we have a plan to change 'PropertyMap' to Structure.
so we have to add new API as structure 'Strikethrough'

Reply:
Done.
Hi @hyunjushin , Please, could you check and let me know if that match your request.